rust/relay/src/server.rs

```diff
@@ -313,7 +314,7 @@ where
         sender: SocketAddr,
         allocation_id: AllocationId,
     ) {
-        let span = tracing::error_span!("peer", %sender, %allocation_id, recipient = field::Empty, channel = field::Empty);
+        let span = tracing::error_span!("handle_relay_input", %sender, %allocation_id, recipient = field::Empty, channel = field::Empty);
```

Comment:
With this span (and the others) we could use the `instrument` macro, I feel it is less noisy, unless there's another reason to enter these explicitly.